pkg/template/generator/expressionvalue.go

```diff
@@ -94,6 +95,8 @@ func replaceWithGenerated(s *string, expression string, ranges [][]byte, length
 			alphabet += Numerals
 		case `\a`:
 			alphabet += Alphabet + Numerals
```

Comment:
appending the same alphabet multiple times means we can unexpected weight generated values... for example `[\w\d\a]` will include numbers three times, and alphabet twice... do we care about that?